src/Microsoft.OData.Client/ClientEdmModel.cs

```diff
@@ -61,6 +71,10 @@ internal ClientEdmModel(ODataProtocolVersion maxProtocolVersion)
             this.maxProtocolVersion = maxProtocolVersion;
             this.EdmStructuredSchemaElements = new ConcurrentEdmSchemaDictionary();
         }
+        internal ClientEdmModel(DataServiceContext context)

```

Comment:
>ClientEdmModel(DataServiceContext context) [](start = 17, length = 42)

inject a "DataServiceContext" into "ClientEdmModel" is not good design. the Model should not internally depend on the context itself. However, we can get the EdmModel using the context. Right?